### ISSUE TITLE
Series.to_csv Future Warning

### ISSUE THREAD

**Issue by jklenzing** [rstoneback/pysat#158]
```
/home/travis/build/rstoneback/pysat/pysat/_files.py:235: FutureWarning: The signature of 
`Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change 
its default value from False to True: please pass an explicit value to suppress this warning.
  date_format='%Y-%m-%d %H:%M:%S.%f')
```

**Comment by jklenzing** [comment]
From the write up at https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.to_csv.html

it sounds like the new default behaviour will output column headers.  Should I add a header=False to maintain, or do we want to switch?

**Comment by jklenzing** [comment]
Proposing we add `header=False` to the `Series.to_csv` statements.

9 instances found in develop branch:
<img width="602" alt="screen shot 2019-02-21 at 9 37 30 pm" src="https://user-images.githubusercontent.com/19592220/53215991-0fe50a00-3621-11e9-99b8-3047dde7d233.png">

**Comment by jklenzing** [comment]
Closed with merge of #173.
